### ISSUE TITLE
Directive template not included in build

### ISSUE THREAD

**Issue by redaikidoka** [Swiip/generator-gulp-angular#982]
Hey, all I have been banging my head against this.

One of my directives (and only one!) is not being included in the build for some reason.
I get a "failed to load resource" and a path to where the template file would be. I can manually fix it by converting the .jade file for my template and putting it in the dist/ folder where the app is looking.

My directive's templateurl:
`         templateUrl: 'app/components/ilpNav/ilpNav.html',
 
and just like my other directives, there is a
`        app/components/ilpNav/ilpNav.jade file, which I presume is being converted to html.

Here's the templateurl for one of my directives that's working:
`       templateUrl: 'app/components/ilpField/ilpField.html',

You can see it's the same format.
That's really where the files are in /src.

Any thoughts?

**Comment by MaxFlower** [comment]
What's in your gulp file. Have You changed something?

**Comment by redaikidoka** [comment]
Nope, the gulp seems to be working awesome, no tinkering for me. And it seems to be the only directive this is happening with (as above, my directive for ilpField is being found just fine).

is there a capitalization thing I need to be concerned about, or is it the fact that it's just included by an undecorated tag, e.g.
<ilp-nav></ilp-nav>

?

**Comment by wstam88** [comment]
Do you begin the templateUrl with or without a forward slash? 
It doesn't seem to work if you begin the templateUrl with a '/'.

**Comment by Swiip** [comment]
Comment the uglify transform in the build and look into the bundle generated for the urls stored in the cache provider. Check also the name of the angular module in which they are added, if you changed the main angular module name, they could be added in a module not loaded.
